examples/caa/caa.ipynb

```diff
@@ -0,0 +1,766 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Contrastive Activation Addition\n",
+    "\n",
+    "This notebook aims to reproduce the workflow defined in [Contrastive Activation Addition](https://arxiv.org/abs/2312.06681) for extracting steering vectors from input. The official codebase can be found [here](https://github.com/nrimsky/CAA)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: steering-vectors in /home/daniel/steering-vectors/.example_caa_venv/lib/python3.10/site-packages (0.5.0)\n",
```

Comment:
It's unfortunate this stuff is all referencing files on your local disk. Maybe we can just delete these outputs from the `outputs` key for the code in Github? It seems like that's what huggingface does for their example notebooks: https://github.com/huggingface/notebooks/blob/main/transformers_doc/en/preprocessing.ipynb

Reply:
yeah i've cleared the outputs in latest commit